test_pull_requests

```diff
@@ -1650,7 +1805,12 @@ elsif mark_test_success_pull_id
 elsif mark_testonlyextended_success_pull_id
   @api_client.mark_test_success(mark_testonlyextended_success_pull_id, pull_id_repo, Properties['settings']['testonlyextended_settings'])
 elsif local_merge_pull_id
-  @api_client.local_merge_pull_request(local_merge_pull_id, pull_id_repo)
+  if ENV['MERGE'].to_s.downcase == 'true'
```

Comment:
Can an ENV var be anything but a string.  Also you can't rely on this env var.  It's not part of tpr.